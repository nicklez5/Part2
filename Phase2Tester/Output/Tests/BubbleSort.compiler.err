Exception in thread "main" java.util.EmptyStackException
	at java.base/java.util.Stack.peek(Stack.java:102)
	at java.base/java.util.Stack.pop(Stack.java:84)
	at J2V.visit(J2V.java:490)
	at J2V.visit(J2V.java:412)
	at J2V.visit(J2V.java:437)
	at J2V.visit(J2V.java:410)
	at J2V.visit(J2V.java:635)
	at J2V.visit(J2V.java:418)
	at J2V.visit(J2V.java:303)
	at J2V.visit(J2V.java:197)
	at J2V.visit(J2V.java:150)
	at J2V.visit(J2V.java:81)
	at J2V.main(J2V.java:28)
