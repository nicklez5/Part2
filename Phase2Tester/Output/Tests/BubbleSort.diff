--- SelfTestCases/BubbleSort.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/BubbleSort.out	2020-02-17 22:35:01.661459697 -0800
@@ -1,22 +1,5 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+--- Error (ProgramFault) ---
+expecting argument 2 to be an integer, found undefined value
+  Main() line 15
+  BBS.Start(<Heap.0>+0 10) line 26
+  BBS.Print(<Heap.0>+0) line 113
