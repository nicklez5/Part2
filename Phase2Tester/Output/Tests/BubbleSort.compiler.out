   const vmt_BBS
      :BBS.Start
      :BBS.Sort
      :BBS.Print
      :BBS.Init

   func Main()
      t.0 = HeapAllocZ(12)
      [t.0] = :vmt_BBS
      if t.0 goto :null0
      Error("null pointer")
      null0:
      t.1 = [t.0]
      t.1 = [t.1+8]
      t.2 = call t.1(t.0 10)
      PrintIntS(t.2)
      ret

   func BBS.Start(this sz)
      t.3 = [this]
      t.3 = [t.3+20]
      t.4 = call t.3(this sz)
      aux01 = t.4
      t.5 = [this]
      t.5 = [t.5+16]
      t.6 = call t.5(this)
      aux01 = t.6
      PrintIntS(99999)
      t.7 = [this]
      t.7 = [t.7+12]
      t.8 = call t.7(this)
      aux01 = t.8
      t.9 = [this]
      t.9 = [t.9+16]
      t.10 = call t.9(this)
      aux01 = t.10
      ret 0

   func BBS.Sort(this)
      t.10 = size
      t.11 = Sub(t.10 1)
      i = t.11
      t.12 = Sub(0 1)
      aux02 = t.12
      t.12 = LtS(aux02 i)
      t.12 = Eq(t.12 1)
      if0 t.12 goto :if5_end
